pom.xml

```diff
@@ -103,20 +105,40 @@
         <artifactId>jsr305</artifactId>
         <version>3.0.2</version>
       </dependency>
-      <dependency>
-        <groupId>org.reactivestreams</groupId>
-        <artifactId>reactive-streams</artifactId>
-        <version>1.0.3</version>
-      </dependency>
       <dependency>
         <groupId>io.projectreactor</groupId>
         <artifactId>reactor-core</artifactId>
         <version>3.4.12</version>
       </dependency>
       <dependency>
-        <groupId>io.projectreactor.netty</groupId>
-        <artifactId>reactor-netty</artifactId>
-        <version>1.0.11</version>
+        <groupId>io.netty</groupId>
```

Comment:
I'd like to understand more about why we are choosing to put this at the global level. It seems like they are only used in the r2dbc folder, wouldn't they be better off there?

Reply:
I fixed this. All deps are next to their usage now.